scripts/upload_image.py

```diff
@@ -18,43 +21,289 @@
 
 Then run `python3 -m pip install cryptography==38.0.4`
 (https://levelup.gitconnected.com/fix-attributeerror-module-lib-has-no-attribute-openssl-521a35d83769)
+
+Usage
+-----
+
+python3 scripts/upload_image.py --help
 """
+
 from __future__ import annotations
 
 import argparse
 import hashlib
+import logging
 import mimetypes
 import os
+import shutil
+import sys
+import tempfile
+from io import BytesIO
+from pathlib import Path
 
+import PIL
```

Comment:
Does this need to be added to `requirements` somewhere? Or mentioned in the docstring at the top?

Reply:
Oh good point! I'll add this to `scripts/requirements-dev.txt` 👍🏻